aip/general/0003.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 3
+state: approved
+created: 2023-03-28
+placement:
+  category: meta
+  order: 30
+---
+
+# AIP Versioning
+
+This AIP defines the versioning scheme of the AIPs.
+
+## Guidance
+
+- The AIPs **must** be versioned by date, using the [ISO-8601 format of `YYYY-MM-DD`](https://www.w3.org/TR/NOTE-datetime), that corresponds to the date the version was added.
+- AIPs versions **must** be available as a tag on the source control mechanism used to store the AIPs, of the format `v{version}`. Example: `v2023-03-28`.
+- The AIPs **should** have a new version every 3 months, or when a significant change to an AIP occurs.
+
+## Appendix
+
+As the AIPs continue to evolve, it is beneficial for
+the AIPs to have a way to denote older versions of the
+specification to serve as a reference point for APIs
+to explain the rationale of their design choices.
```

Comment:
Combine the two paragraphs to read,

> As AIPs evolve, versioning is beneficial. Versions serve as reference points that demonstrate the rationale for design choices. They are crucial since sometimes the guidance within an AIP is reversed or includes updates to design patterns that may not look similar to the original design.

Reply:
I'm confused by your response. You talk about a singular "paragraph". I am suggesting combing this paragraph with the one below, thus eliminating the second paragraph.

In fact, in my edit, s/are crucial since/are crucial as/.

Less is more. No meaning is lost by combining the paragraphs and the readability is improved.

Reply:
ack, I agree the paragraphs are more verbose than they need to be. I'll push an update that shrinks to a single paragraph while keeping a point I think is valuable (clients referencing older versions).

Reply:
PTAL.